dev/src/main/java/org/eclipse/codewind/intellij/ui/constants/UIConstants.java

```diff
@@ -14,9 +14,9 @@
     // Documentation links
     public static final String
             DOC_BASE_URL = "https://www.eclipse.org/codewind",
-            GETTING_STARTED_INFO_URL = DOC_BASE_URL + "/mdt-intellij-getting-started.html",
-            CWSETTINGS_INFO_URL = DOC_BASE_URL + "/mdteclipsemanagingprojects.html",
-            TEMPLATES_INFO_URL = DOC_BASE_URL + "/mdteclipseworkingwithtemplates.html",
-            REGISTRY_INFO_URL = DOC_BASE_URL + "/image-registry-credentials.html",
-            REMOTE_SETUP_URL =  DOC_BASE_URL +"/remoteconfiguring.html";
+            GETTING_STARTED_INFO_URL = DOC_BASE_URL + "/intellij-getting-started.html",
+            CWSETTINGS_INFO_URL = DOC_BASE_URL + "/project-actions.html",
+            TEMPLATES_INFO_URL = DOC_BASE_URL + "/workingwithtemplates.html",
+            REGISTRY_INFO_URL = DOC_BASE_URL + "/che-setupregistries.html",
```

Comment:
Ask team if we need to add an IntelliJ specific image registry credentials file.

Reply:
There needs to be an image registry credential file shared by all the IDEs